types/happypack/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for happypack
+// Project: https://github.com/amireh/happypack
+// Definitions by: Akash Vishwakarma <https://github.com/akashishu777>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import { Plugin } from 'webpack';
+
+declare class HappyPack extends Plugin {
+
+    constructor(options?: HappyPack.PluginOptions);
```

Comment:
The docs and source seem to indicate that this is a should be a required parameter.

Reply:
options are now required.

Reply:
I have updated the code now options are required.